README.md

```diff
@@ -8,34 +8,318 @@ or
+end
+```
+
+#### Design Action
+```
+def design
+  @page ||= params['page'].try(:to_i) || 1
+  store_url = "stores/#{params[:id]}?page=#{@page}"
+  store_url += "&album_id=#{params['album_id']}" if params['album_id'].present?
+  store_url += "&product_type=#{params['product_type']}" if params['product_type'].present?
+
+  fetch_store = TeePublic::Api.send(store_url).deep_symbolize_keys!
+  @store = fetch_store[:body]
+
+  fetch_design = TeePublic::Api.designs(params['design_id']).deep_symbolize_keys!
+  @design = fetch_design[:body]
+
+
+  skus_url = "designs/#{params['design_id']}/products/#{params['product_type']}"
+  fetch_skus = TeePublic::Api.send(skus_url).deep_symbolize_keys!
+  @skus = fetch_skus[:body]
+
+  # Set at the end since this requires a response.
+  @affiliate_network_id = fetch_store.dig(:headers, :'x-affiliatenetwork-id')
+end
 ```
-import { DesignCollection } from 'teepublic-react'
 
-class MyStore extends React.Component {
+#### Cart action
+```
+def cart
+end
+```
+
+## Step 7 - Creating views
+Create 3 corresponding views to actions added in step 6
+
+#### Store view
+Add store.html.erb
+```
+<%= react_component('TeePublicStore', store: @store, page: @page, albumId: params[:album_id], productTypeName: params[:product_type]) %>
+```
+
+#### Design view
+Add design.html.erb
+```
+<%= react_component('TeePublicDesign', store: @store, design: @design, skuData: @skus, affiliateNetworkId: @affiliate_network_id) %>
+```
+
+#### Cart view
+Add cart.html.erb
+```
+<%= react_component('TeePublicCart') %>
+```
+
+## Step 8 - Creating TeePublic React Components
+Let's install the latest version of teepublic-react package
+
+`npm i teepublic-react`
+
+import teepublic-react css to your application
+
+`import 'teepublic-react/build/css/index.css'`
+
+Component level documentation could be found below in documentation section.
+
+As you noticed in the Step 7, we used some components (`TeePublicStore`, `TeePublicDesign`, `TeePublicCart`) which doesn't exist.
+#### `<TeePublicStore />`
+Add this new component to your react component library.
+Copy paste the code below with your necessary tweaks
+
+```
+import React from 'react'
+
+import { Footer, Store } from 'teepublic-react'
+// We will talk about this in Step 9
+import { TeePublicRoutes } from '../TeePublicConfiguration'
+
+export default class TeePublicStore extends React.Component {
+  render () {
+    const { store, page, albumId, productTypeName  } = this.props
+
+    return (
+      <div className='teepublic store'>
+        <Store
+          className='contain'
+          storeData={store}
+          configuration={TeePublicRoutes}
+          selectedAlbumId={albumId}
+          selectedPage={page}
+          selectedProductTypeName={productTypeName || ''}
+        />
+        <Footer />
+      </div>
+    )
+  }
+}
+```
+
+#### `<TeePublicDesign />`
+Add this new component to your react component library.
+Copy paste the code below with your necessary tweaks
```

Comment:
If you find yourself just copy/pasting notes, they're probably not needed.

Reply:
ok cool